### PR TITLE
Use maildump instead of ConsoleEmailHandler

### PR DESCRIPTION
**Pull request by burhan**
In development, the `ConsoleEmailHandler` is of limited use if you are sending emails with attachments, or any HTML message.

I propose integrating with [maildump](https://github.com/ThiefMaster/maildump) a Python clone of [mailcatcher](http://mailcatcher.me/) which is a ruby application.

It provides a local SMTP server listening on 1025, and an interface that shows multi-part emails and HTML formatted emails correctly, available at http://127.0.0.1:1080:

image

The way I use it is through two grunt tasks, `start-email-server` and `stop-email-server`.

### CODE REVIEW

**Comment by pydanny** [comment]
Do emails still send if `start-email-server` is not yet started?

**Comment by burhan** [comment]
No, you get an exception "Connection refused".

**Comment by pydanny** [comment]
@burhan Could you make this an available option during project creation?

**Comment by burhan** [comment]
@pydanny - done.

File: {{cookiecutter.repo_name}}/requirements/local.txt [hunk 1]
```diff
@@ -10,3 +10,8 @@ django-debug-toolbar==1.3.2
 
 # improved REPL
 ipdb==0.8.1
+
+{% if cookiecutter.use_maildump == "y" -%}
+# maildump
```
Comment (theskumar) [inline {{cookiecutter.repo_name}}/requirements/local.txt]: Shall we call this comment as `# Email Debugging`

File: {{cookiecutter.repo_name}}/README.rst [hunk 2]
```diff
@@ -126,6 +126,28 @@ To run a celery worker:
...
+
+This will start an email server. The project is setup to deliver to the email server by default. To view messages
+that are sent by your application, open your browser to http://127.0.0.1:1080
+
+To stop the email server::
+
+    $ grunt stop-email-server
+
+The email server listens on 127.0.0.1:1025
+{% endif %}
```
Comment (theskumar) [inline {{cookiecutter.repo_name}}/README.rst]: +1

File: {{cookiecutter.repo_name}}/requirements/local.txt (hunk 1 above)
Comment (burhan) [inline {{cookiecutter.repo_name}}/requirements/local.txt]: Agreed, made it the same as the master README.

**Pull request merged**